rasa/shared/core/domain.py

```diff
@@ -1860,7 +1860,11 @@ def slot_mapping_for_form(self, form_name: Text) -> Dict[Text, Any]:
         Returns:
             The slot mapping or an empty dictionary in case no mapping was found.
         """
-        return self.forms.get(form_name, {})[REQUIRED_SLOTS_KEY]
+        form = self.forms.get(form_name)
+        if form:
+            return form.get(REQUIRED_SLOTS_KEY)
```

Comment:
This will return `None` if there is no `REQUIRED_SLOTS_KEY` - should it be an empty dict?

Reply:
That's the interesting bit:  when the form has custom slot mappings, but without the `REQUIRED_SLOTS_KEY` explicitly mentioned in `domain.yml`, the form will contain this key because of the transformation that happens in [here](https://github.com/RasaHQ/rasa/blob/f1e19f5e2853880873bbfcec828fb54b7ccb12bf/rasa/shared/core/domain.py#L707). Do you still think anything should be changed in these conditions?

Reply:
ah ok yes I see. Then maybe `form[REQUIRED_SLOTS_KEY]` is more explicit?